test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/project.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
     "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-004056",
-    "Microsoft.DotNet.InternalAbstractions": "1.0.0",
+    "Microsoft.DotNet.InternalAbstractions": "1.0.500-preview2-1-003177",
```

Comment:
When are we going to revert this? (Or how will we remember to revert it?)  I guess `-*` doesn't work?